src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -204,33 +115,9 @@ public class Graph implements Serializable {
    * If this graph contains elevation data, the maximum value.
    */
   public Double maxElevation = null;
-  /** Parent stops **/
-  public Map<FeedScopedId, Station> stationById = new HashMap<>();
-  /**
-   * Optional level above parent stops (only supported in NeTEx)
-   */
-  public Map<FeedScopedId, MultiModalStation> multiModalStationById = new HashMap<>();
-  /**
-   * Optional grouping that can contain both stations and multimodal stations (only supported in
-   * NeTEx)
-   */
-  public Map<FeedScopedId, GroupOfStations> groupOfStationsById = new HashMap<>();
-  /**
-   * TripPatterns used to be reached through hop edges, but we're not creating on-board transit
-   * vertices/edges anymore.
-   */
-  public Map<FeedScopedId, TripPattern> tripPatternForId = Maps.newHashMap();
-  public Map<FeedScopedId, TripOnServiceDate> tripOnServiceDates = Maps.newHashMap();
-  public Map<FeedScopedId, FlexStopLocation> locationsById = new HashMap<>();
-  public Map<FeedScopedId, FlexLocationGroup> locationGroupsById = new HashMap<>();
-  public Map<FeedScopedId, FlexTrip> flexTripsById = new HashMap<>();
+
   /** The distance between elevation samples used in CompactElevationProfile. */
   private double distanceBetweenElevationSamples;
```

Comment:
We might want to remove this altogether from the graph object, and instead always serialize directly from and to the static variable in `CompactElevationProfile` in `SerializedGraphObject`

Reply:
TODO added for subsequent refactoring